### PR TITLE
Add guide for migrating the rpool

### PR DESCRIPTION
**Pull request by mir07**
Signed-off-by: Michael Rasmussen <mir@datanom.net>

### CODE REVIEW

File: _docs/info/migrate_rpool.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+After detaching the old disk(s) from the rpool you should do a reboot to ensure everything is fine.
+
+When rebooting you might see a warning from 'devid register' that device ids has changed when removing the old disk(s). This is just a warning which will be fixed by creating a new boot_archive either manually or automatically by OmniOSce when you reboot or power down your system.
+
+## Contact
+
+If you have any problems or questions, please get in touch via
+[the lobby](https://gitter.im/omniosorg/Lobby) or
+[#omnios on Freenode](http://webchat.freenode.net?randomnick=1&channels=%23omnios&uio=d4)
+
```
Comment (citrus-it) [inline _docs/info/migrate_rpool.md:122]: Can you replace this with:

```
Any problems or questions, please [get in touch](/about/contact.html).
```

which takes them to the contact page and centralises the various contact methods.
Thanks

File: _docs/info/migrate_rpool.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+## Attach the new disk(s) to rpool
+
+The first step is to attach the new disks to the existing rpool using the command **zpool attach** like this:
+
+``` terminal
+$ sudo zpool attach rpool c1t1d0 c1t3d0
+Password: 
+Make sure to wait until resilver is done before rebooting.
+```
+
```
Comment (citrus-it) [inline _docs/info/migrate_rpool.md:77]: Could you add the `zpool status` output here so people can see what this attach command did to the pool layout?

File: _docs/info/migrate_rpool.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+If you were able to make a successful reboot on the new disk(s) then the time has come to detach the old disk(s) from the rpool using the command **zpool detach** like this:
+
+``` terminal
+sudo zpool detach rpool c1t1d0
+sudo zpool detach rpool c1t2d0
+```
+
+After detaching the old disk(s) from the rpool you should do a reboot to ensure everything is fine.
+
```
Comment (oetiker) [inline _docs/info/migrate_rpool.md:138]: I would add a note about pool auto resizing if the new disks are larger than the old ones ... 

```
zpool set autoexpand=on rpool
```

Comment (mir07) [reply _docs/info/migrate_rpool.md:138]: Added information in the latest commit.

**Comment by oetiker** [comment]
thanks for your contrib!

**Pull request merged**